lib/guardian.rb

```diff
@@ -559,8 +559,7 @@ def can_ignore_user?(target_user)
 
   def can_ignore_users?
     return false if anonymous?
-    @user.staff? || @user.has_trust_level?(SiteSetting.min_trust_level_to_allow_ignore.to_i) ||
-      @user.in_any_groups?(SiteSetting.ignore_allowed_groups_map)
+    @user.staff? || @user.in_any_groups?(SiteSetting.ignore_allowed_groups_map)
```

Comment:
Removing `SiteSetting.min_trust_level_to_allow_ignore` is what makes the deprecation notice go away 💨